RELEASE/scripts/autoscend/auto_quest_level_11.ash

```diff
@@ -210,6 +210,13 @@ boolean L11_aridDesert()
 	{
 		string temp = visit_url("place.php?whichplace=desertbeach", false);
 	}
+	
+	// TODO: Mafia currently (r20019) does not properly track desert exploration progress in plumber
+	// Not sure if this fix can do exact progress, but when I visited that page it corrected my explortion to 100 and quest progress to done so it will prevent adv loss at least
+	if(in_zelda() && get_property("desertExploration").to_int() < 100)
+	{
+		string discard = visit_url("place.php?whichplace=desertbeach");
```

Comment:
if you're not going to use the return value you can just omit it in future. Not an issue here as it'll immediately go out of scope and should be deleted/garbage collected (however ASH deals with that, I have no idea) but keep this in mind in the future.

Reply:
It is there to stop mafia from exploding the gCLI with a 2 page display of the HTML of the desert every time you visit it.
type into CLI
`ash visit_url("place.php?whichplace=desertbeach");`
and you will see why I put it there.

Reply:
> It is there to stop mafia from exploding the gCLI with a 2 page display of the HTML of the desert every time you visit it.
> type into CLI
> `ash visit_url("place.php?whichplace=desertbeach");`
> to see what I am talking about

These are not the same thing. visit_url in an ash script without a return value doesn't behave the same way as the ash command on the CLI. For very good reason.

Reply:
Ash doesn't behave the same as a script?
Well that is good to know, i will stop using it for simulation purposes and instead use a test script